modules/concurrency/tests/spinner_tests.cpp

```diff
@@ -50,34 +37,37 @@ TEST(SpinnerTest, SpinTest) {
   future.get();
 
   // The counter should have been incremented.
-  EXPECT_GT(spinner.counter.load(), 0);
+  EXPECT_GT(spinner.spinCount(), 0);
 }
 
-TEST(SpinnerTest, StopCallbackTest) {
-  TestSpinner spinner;
-  std::atomic<bool> callback_called(false);
+TEST(SpinnerTest, StopCallback) {
+  static constexpr auto WAIT_FOR = std::chrono::milliseconds{ 10 };
 
-  spinner.addStopCallback([&]() { callback_called.store(true); });
+  std::size_t callback_called_counter = 0;
+  Spinner spinner([&callback_called_counter] { ++callback_called_counter; });
```

Comment:
nit: interesting didn't know you could omit the () ... maybe still add them? for me this just looks strange ;)